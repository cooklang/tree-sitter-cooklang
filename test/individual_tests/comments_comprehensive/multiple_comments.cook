-- First comment
Stir well [- remember to scrape bottom -] -- another comment