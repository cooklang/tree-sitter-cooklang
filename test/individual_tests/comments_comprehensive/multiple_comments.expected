(recipe
  (comment)
  (step
    (plain_text)
    (block_comment)
    (plain_text)
    (comment)))